dao/src/main/java/greencity/entity/Habit.java

```diff
@@ -48,6 +48,12 @@ public class Habit {
     @Column(name = "default_duration", nullable = false)
     private Integer defaultDuration;
 
+    @Column(name = "is_custom_habit", nullable = false)
+    private Boolean isCustomHabit;
+
+    @Column(name = "user_id")
```

Comment:
can user_id be `null`?

Reply:
Yes, these fields will not be null for the custom habit in order to know who create  custom habit, but null for the default one.